api/models/metadata.go

```diff
@@ -18,34 +18,47 @@ import (
 	"database/sql/driver"
 	"encoding/json"
 	"errors"
-	"fmt"
 
 	"github.com/gojek/merlin/mlp"
 	"github.com/gojek/merlin/utils"
 )
 
 const (
-	labelTeamName         = "gojek.com/team"
-	labelStreamName       = "gojek.com/stream"
-	labelAppName          = "gojek.com/app"
-	labelOrchestratorName = "gojek.com/orchestrator"
-	labelEnvironment      = "gojek.com/environment"
-	labelUsersPrefix      = "gojek.com/%s"
+	LabelAppName          = "app"
+	LabelComponent        = "component"
```

Comment:
do we need to expose this const?

Reply:
Currently, only LabelOrchestratorName that used by other packages, so yeah, we will change other to local variables.